src/fee-accounts/store/effects/fee-accounts.effects.ts

```diff
@@ -23,9 +23,10 @@ export class FeeAccountsEffects {
     switchMap((payload: any) => {
       return this.feeAccountsService.fetchFeeAccounts(payload.paymentAccounts).pipe(
         map(feeAccountsDetails => new feeAccountsActions.LoadFeeAccountsSuccess(feeAccountsDetails)),
-        catchError(error => {
-          this.loggerService.error(error.message);
-          return of(new feeAccountsActions.LoadFeeAccountsFail(error));
+        catchError(errorResponse => {
+          this.loggerService.error(errorResponse);
```

Comment:
there is something really odd going on here
this should be split is separate effects